src/notation/view/notationpaintview.cpp

```diff
@@ -78,6 +90,30 @@ NotationPaintView::~NotationPaintView()
     delete m_noteInputCursor;
 }
 
+void NotationPaintView::requestScrollHorizontal(qreal position)
```

Comment:
Please, rename this method to scrollHorizontal (along with the method placed bellow)